ci/ci/github.py

```diff
@@ -821,7 +821,7 @@ def config(self):
             config.build(deploy_batch, self, scope='dev')
             deploy_batch = await deploy_batch.submit()
             self.deploy_batch = deploy_batch
-            await deploy_batch.wait()
+            return await deploy_batch.id()
```

Comment:
I don't think this should be a method. `deploy_batch.id`

Reply:
You're right, thanks.